xdsl/tools/command_line_tool.py

```diff
@@ -86,6 +86,11 @@ def get_convert_stencil_to_ll_mlir():
 
         return convert_stencil_to_ll_mlir.ConvertStencilToLLMLIRPass
 
+    def get_jax_use_donated_arguments():
+        from xdsl.transforms.experimental import jax_use_donated_arguments
```

Comment:
I would recommend against experimental, it's a place for untested code, that we will probably remove at some point, let's just put it in the main transforms folder